mteb/tasks/Clustering/dan/WikiClusteringP2P.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
```

Comment:
I would probably either add a script on dataset construction (see the scripts folder) as well as a text description of how the dataset is constructed.

Reply:
Good idea, I'll probably link to the repo I've created for generating the dataset and use that as docs:))